test/local-fs.spec.ts

```diff
@@ -114,6 +115,14 @@ describe(`the local filesystem implementation`, () => {
                 .then(() => expect(fs.loadTextFile(fileName)).to.eventually.equals(newContent));
         });
 
+        it(`ignores events from ignored dir in watcher`, () => {
```

Comment:
I'm missing tests for when 'fileSystemError' actually happens - This here is a test that verifies this event does *not* happen, which will pass even if at some point we delete the event's implementation from the code.

There is no place for an API feature that can't be triggered.